js/templates/search/search.html

```diff
@@ -79,13 +80,23 @@ <h4 class="rowSm"><%= val.label %></h4>
                 </div>
               <% } else if (val.type ==='checkbox') { %>
                 <div class="flexCol">
-                <% _.each(val.options, function(option, index) {
-                  let checked = '';
-                  checked = option.checked ? 'checked' : '';
-                  var parsedLabel = ob.parseEmojis(option.label);
-                  print(`<input type="checkbox" ${checked} id="${option.name}" name="${option.name}">`);
-                  print(`<label for="${option.name}">${parsedLabel}</label>`);
-                }); %>
+                <%
+                  // find all the values that should be checked
+                  const checkedVals = new Set();
+                  val.options.forEach(opt => {
+                    if (opt.checked) checkedVals.add(opt.value);
+                  });
+                  val.options.forEach((option, index) => {
```

Comment:
Why the second separate forEach, instead of just adding this code to the first one?

Reply:
Because first it's finding out if any of the options are checked. The second loop only applies the default if none of the options are checked.

It could be done more concisely with a filter.